<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDlg</class>
 <widget class="QDialog" name="SettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAudio">
      <attribute name="title">
       <string>&amp;Audio</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Input device</string>
         </property>
         <property name="buddy">
          <cstring>comboAudioInputDevice</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboAudioInputDevice"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Output device</string>
         </property>
         <property name="buddy">
          <cstring>comboAudioOutputDevice</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboAudioOutputDevice"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Samplerate</string>
         </property>
         <property name="buddy">
          <cstring>comboAudioSamplerate</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboAudioSamplerate">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="buttonAsioPanel">
         <property name="text">
          <string>Show ASIO &amp;control panel</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="checkAsioPanelClosesDevice">
         <property name="text">
          <string>Close audio device when showing ASIO control panel</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMultithreading">
      <attribute name="title">
       <string>&amp;Threads</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabMidi">
      <attribute name="title">
       <string>&amp;MIDI</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
